app.rb

```diff
@@ -55,6 +55,10 @@ def legislatures_in_pr(payload)
     pr_files = Octokit.pull_request_files(payload['repository']['full_name'], payload['number'])
     pr_files.map { |f| legislature_for_file(f['filename']) }.compact.uniq
   end
+
+  def legislatures_for_sources(sources)
+      sources.map { |s| EveryPolitician::Legislature::from_sources_dir(s.to_s) }.map { |l| { :country_slug => l.country.slug, :legislature_slug  => l.slug } }
```

Comment:
I think this should probably be two space indentation for consistency. It would also be good to use a `.` rather than `::` between the class and method 😉